docs/source/resources/define-metadata.rst

```diff
@@ -1965,6 +1960,11 @@ The use of the Python version selectors, `py27`, `py34`, etc. is discouraged in
 favor of the more general comparison operators.  Additional selectors in this
 series will not be added to conda-build.
 
+Note that for each subdir with OS and architecture that `conda` supports,
+two preprocessing selectors are created for the OS and the architecture separately
+except when the architecture is not a valid python expression (`*-32` and `*-64`
+in particular).
+
```

Comment:
Not released yet (https://github.com/conda/conda-build/commit/d480c32597d7cb986ec89dbc8152d474b627e404)